app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="navigation py-6 px-3 blue-nav-white-text">
+  <ul class="flex justify-between items-center">
+    <li class="w-10">
+      <%= link_to image_tag('backarrow.png', alt: 'Back'), root_path, class: 'nav-link' %>
+    </li>
+    <h2><strong>Sign up</strong></h2>
+    <li>
+      <%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: "invisible" %>
+    </li>
+  </ul>
+</div>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { class: 'mt-10 flex flex-col' }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field border-t-2 ">
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", class: 'w-full  border-none', placeholder: 'Name' %>
+  </div>
+
+  <div class="field border-t-2">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'w-full border-none', placeholder: 'Email' %>
+  </div>
+
+  <div class="field border-t-2">
+    <%= f.password_field :password, autocomplete: "new-password", class: 'w-full  border-none', placeholder: 'Password (6 characters minimum)' %>
+  </div>
+
+  <div class="field border-y-2">
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", class: 'w-full border-none', placeholder: 'Password confirmation' %>
+  </div>
+
+  <div class="actions ms-4">
+    <%= f.submit "Sign up", class: 'mt-6 p-3 px-5 green-btn rounded hover:bg-green-600' %>
+  </div>
+<% end %>
```

Comment:
- [ ] Kindly ensure your design follows the given design for the signup page and the login page. Now since the given design only has the `Registration` page, this is the one that we have to customize for our `Login` page as well.

Your design
![image](https://github.com/VaheStepanyan100/budget-app/assets/81932478/2c163525-afc0-4db2-aaa7-870c2e06df8e)

Given design
![image](https://github.com/VaheStepanyan100/budget-app/assets/81932478/f2a58071-d6b5-450c-ad66-bfbad53f1f43)

- The signup button should be a link in white at the top to the right as is in the given design 

#### Note that this change applies to the login page as well

Reply:
This change has been implemented well